fvm/src/gas/price_list.rs

```diff
@@ -244,6 +357,17 @@ pub struct PriceList {
     pub(crate) verify_post_discount: bool,
     pub(crate) verify_consensus_fault: i64,
     pub(crate) verify_replica_update: i64,
+    // 1 Fuel = gas_per_fuel * 1 Gas
+    pub(crate) gas_per_fuel: i64,
+
+    pub(crate) extern_traversal_cost: i64,
+    // TODO: Some of these will probably need per-byte multipliers
+    pub(crate) block_open: i64,
```

Comment:
This will need a per byte multiplier since the open will retain the block content in Rust memory.